vms/platformvm/block/builder/builder.go

```diff
@@ -123,38 +123,22 @@ func (b *builder) buildBlock() (block.Block, error) {
 		return nil, err
 	}
 	nextHeight := preferred.Height() + 1
+
 	preferredState, ok := b.blkManager.GetState(preferredID)
 	if !ok {
 		return nil, fmt.Errorf("%w: %s", state.ErrMissingParentState, preferredID)
 	}
 
-	timestamp := b.txExecutorBackend.Clk.Time()
```

Comment:
The logic to pick next block time is duplicated here and it the mempool tx verification. This change removes the duplication